tck/LICENSE_EFTL.md

```diff
@@ -0,0 +1,83 @@
+# Eclipse Foundation Technology Compatibility Kit License - v 1.0
+
+Copyright (c) 2018, Eclipse Foundation, Inc. and its licensors.
```

Comment:
should the `2018` be changed to `2022` to reflect this file being added in 2022?